LEAF_Request_Portal/form.php

```diff
@@ -3048,14 +3053,32 @@ public function query($inQuery)
                         AND deleted = 0', array());
             foreach ($res2 as $item)
             {
-                if($data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] == null
-                    || $data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] < $item['fulfillmentTime']) {
+                if($data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] == null || $data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] < $item['fulfillmentTime']) {
                     $data[$item['recordID']]['recordResolutionData']['lastStatus'] = $item['lastStatus'];
                     $data[$item['recordID']]['recordResolutionData']['fulfillmentTime'] = $item['fulfillmentTime'];
                 }
             }
         }
 
+        if($joinRecordResolutionBy) {
+            require_once 'VAMC_Directory.php';
+            $dir = new VAMC_Directory;
+
+            $res2 = $this->db->prepared_query('SELECT recordID, stepID, userID, time, description, actionTextPasttense, actionType, comment FROM action_history
+    											LEFT JOIN dependencies USING (dependencyID)
+    											LEFT JOIN actions USING (actionType)
+    											WHERE recordID IN (' . $recordIDs . ')
```

Comment:
This needs to use the method the other sql statements use.  Inside the statement itself, use the template variable style (ex :recordsIDs) and give they value in the array fed to the prepared_query method.